tqp/flask.py

```diff
@@ -1,10 +1,12 @@
 import logging
 
 import flask
-from flask import _app_ctx_stack as context_stack
```

Comment:
Removed in flask 3.x. See https://flask.palletsprojects.com/en/3.0.x/changes/#version-2-2-0